dts/arm/xilinx/zynqmp.dtsi

```diff
@@ -125,7 +122,13 @@
 			tx-buffer-size = <512>;
 			discard-rx-fcs;
 			unicast-hash;
-			full-duplex;
+		};
+
+		gem0_mdio: gem0_mdio {
```

Comment:
In Linux world DT node names shouldn't use _ but should use - instead. 
Also see this error message when used.
/tmp/kv260_r5/zephyr/zephyr.dts:149.24-154.5: Warning (simple_bus_reg): /soc/gem0_mdio: missing or empty reg/ranges property

Reply:
Zephyr's DT generation tools produce a syntax error when using dashes instead of underscores.